d3dcore/src/d3d_info.cpp

```diff
@@ -0,0 +1,12 @@
+#include <assert.h>
+#include <iostream>
+#include <string>
+
+#include "d3dcore/d3d_info.h"
```

Comment:
#include "file.h" assumes it is located in the same directory, which is not true in our case